Sources/PointFree/Home.swift

```diff
@@ -124,3 +143,25 @@ private let episodeInfoColumnView = View<Episode> { ep in
     ]
   )
 }
+
+private let blueGradientClass = CssSelector.class("blue-gradient")
+private let blueGradientStyles =
+  blueGradientClass % (
+    key("background", "rgba(128,219,255,1)")
+      <> key("background", "-moz-linear-gradient(top, rgba(128,219,255,1) 0%, rgba(128,219,255,0) 100%)")
```

Comment:
we dont have first class support for gradients or transforms in our css lib, so just doing raw css for now